seed_db.sh

```diff
@@ -0,0 +1,136 @@
+# ***** How to Use: *****
+# For now, you will need to set the two secret variables $USER_AUTH_ID and $ADMIN_AUTH_ID in your own shell.
+# The values of these variables can be found in the secret channel. 
+# Working to add these to vault and, in a future update, you won't need to set these variables manually. 
+# 
+# In the directory of the script, call ./seed_db.sh [OPTION] 
+#
+# [OPTIONS]: 
+#   erase - erases the contents of the database. To avoid conflicting entries, 
+#           please call this to erase the contents of the db. Note that this 
+#           will erase all of the contents in the story_translations, users, 
+#           and stories databases. 
+# 
+#   kevin - generates story contents that, to say the least, are probably 
+#           not NGO-friendly. Please use this option cautiously. 
+
+
+
+if [ -z "$USER_AUTH_ID" ] 
+then 
+    echo "Warning: \$USER_AUTH_ID is empty. Please set the value to the one posted in the secrets channel." 
+fi 
+
+if [ -z "$ADMIN_AUTH_ID" ] 
```

Comment:
i think the script should exit if these AUTH_IDs are missing, unless the `erase` command is being passed in.